### ISSUE TITLE
handler in SensuWriterFactory 

### ISSUE THREAD

**Issue by iakovgan** [jmxtrans/jmxtrans#544]
Hi, 
how can i specify handler to SensuWriterFactory class? 

deprecated SensuWriter has this parameter bu it does not have rootPrefix

Regards
Iakov

**Comment by gehel** [comment]
Yep, that's something that is not supported yet in the new SensuWriterFactory. It should not be all that hard to add. Feel free to send a pull request...
